sleepylyze/ioeeg.py

```diff
@@ -137,12 +137,12 @@ def get_info(self):
         print("Patient identifier:", self.metadata['in_num'])
 
         # read the first line to check encoding
-        with open(self.metadata['filepath'], 'r') as f:
+        with open(self.metadata['filepath'], encoding='utf16') as f:
             line = f.readline()
         if line[3] == '\x00':
             f_encoding = 'utf-16-le'
         else:
```

Comment:
```suggestion
        # try opening the file without specifying encoding
        try:
            with open(self.metadata['filepath'], 'r') as f:
                line = f.readline()
            if line[3] == '\x00':
                f_encoding = 'utf-16-le'
            else:
                f_encoding = 'ascii'
        # if we can't open the file without specifying the encoding, assume it's utf16
        except UnicodeDecodeError:
```

Reply:
Great suggestion. This works on my machine with both the old Schiff Lab files and the shhs files text files.